source/Government.cpp

```diff
@@ -204,12 +233,21 @@ double Government::InitialPlayerReputation() const
 
 
 // Get the amount that your reputation changes for the given offense.
-double Government::PenaltyFor(int eventType) const
+double Government::PenaltyFor(int eventType, bool onDrone) const
 {
 	double penalty = 0.;
-	for(const auto &it : penaltyFor)
-		if(eventType & it.first)
-			penalty += it.second;
+	if(!onDrone)
+	{
+	    for(const auto &it : penaltyFor)
+            if(eventType & it.first)
+                penalty += it.second;
+    }
+	else
+	{
+        for(const auto &it : dronePenaltyFor)
+            if(eventType & it.first)
+                penalty += it.second;
+    }
```

Comment:
The suggestion from my other review comment would remove this anyway, but for future reference, use tabs, not spaces.

Reply:
sorry thing is my university literally told me the opposite so I have to change some settings ^^' forgot some
guess I'll install a new editor from fresh (notepadd++ good ?) and use editorconfig to be sure

Reply:
As a general comment, I know a lot of people working on ES use NP++. There's even a custom language file floating around somewhere for it, but that'll only help on the content creation side, not the code writing.

Reply:
Well really the standard is to match the coding style of whatever project you're working on, in which case we use tabs, :)